ops/services/app_service/main.tf

```diff
@@ -92,11 +94,13 @@ resource "azurerm_linux_web_app_slot" "staging" {
 
   # Configure Docker Image to load on start
   site_config {
-    always_on               = "true"
-    scm_minimum_tls_version = "1.2"
-    use_32_bit_worker       = false
-    ftps_state              = "Disabled"
-    vnet_route_all_enabled  = false
+    always_on                         = "true"
+    health_check_path                 = "/health"
```

Comment:
I don't believe this path will work. Taken from the following: https://learn.microsoft.com/en-us/azure/app-service/monitor-instances-health-check?tabs=dotnet

> Note that /api/health is just an example added for illustration purposes. We do not create a Health Check path by default. You should make sure that the path you are selecting is a valid path that exists within your application

Reply:
@alismx Do you have documentation to the contrary?

Reply:
I've selected this path because it's an endpoint that we have in our app, https://github.com/CDCgov/prime-simplereport/blob/309e5b9fe82c7876ea1c31c862ddc09421ab6961/backend/src/main/java/gov/cdc/usds/simplereport/config/WebConfiguration.java#L19

Reply:
@rin-skylight Does this address the concern you had? Azure doesn't create a path for us, but we do have one available to us already.

Reply:
...yep, I messed up on this one. I didn't realize this path returned a valid 200 code...I thought we would have needed to go through the actuator endpoint. I will gladly own that mistake; thank you for clarifying!

Reply:
No worries! Thank you for taking a look! :smile: